prow/plank/reconciler.go

```diff
@@ -102,6 +102,10 @@ func add(
 
 	r := newReconciler(ctx, mgr.GetClient(), overwriteReconcile, cfg, opener, totURL)
 	for buildCluster, buildClusterMgr := range buildMgrs {
+		r.log.WithFields(logrus.Fields{
```

Comment:
This one isn't necessary but would be helpful in debugging a similar issue with kubeconfigs.